src/main/java/madstax/view/NavigationBar.java

```diff
@@ -53,7 +54,7 @@ private void createNavBarDefaultItems() {
         backButton.setAlignmentX(SwingConstants.LEFT);
         backButton.setVisible(false);
 
-        JLabel titleLabel = new JLabel("Role Selection");
+        JLabel titleLabel = new JLabel("Title");
```

Comment:
Is it worth making "Title" a constant like you did within the CoursePlannerScreen?

Reply:
Removed!